includes/API/GetServerInfoHandler.php

```diff
@@ -8,7 +8,7 @@
 class GetServerInfoHandler extends SimpleHandler {
 	/** @inheritDoc */
 	public function run(): array {
-		$apiVersion = MediaWikiServices::getInstance()->getMainConfig()->get( 'DAAPIVersion' );
+		$apiVersion = (string) MediaWikiServices::getInstance()->getMainConfig()->get( 'DAAPIVersion' );
```

Comment:
Is it better to do this than doing `/* @var string */`? I tried using the `@var` but it wasn't detected by psalm.